Project/Back-End/job_posting_app/serializers.py

```diff
@@ -5,4 +5,4 @@ class Job_PostingSerializer(ModelSerializer):
 
     class Meta:
         model = Job_Posting
-        fields = ["__all__".lower()]
\ No newline at end of file
+        fields = ["title".lower(), "job_type".lower(), "job_description", "degree_type".lower(),"skill","salary","location","applicants","company", "recruiter"]
```

Comment:
I'm not sure what the purpose of `.lower()` is here. I may be mistaken but it appears that the `.lower()` function is applying to a hard-coded string in each case, all of which are already completely lowercase. Are you sure `.lower()` is doing anything here?